package.json

```diff
@@ -29,45 +29,32 @@
         "url": "https://github.com/inclusive-design/codesign.inclusivedesign.ca/issues"
     },
     "homepage": "https://github.com/inclusive-design/codesign.inclusivedesign.ca#readme",
-    "husky": {
-        "hooks": {
-            "pre-commit": "npm run lint",
-            "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
You'll need to create a new file for this one too, @chosww

Reply:
In the example on the doc, both pre-commit and commit-msg are inside of a file called commit-msg, so I assumed that they can be written in a single file. Are you suggesting that we should create a separate file for these two commands?

Reply:
The scripts we want to run happen on two different hooks, `pre-commit` and `commit-msg`, so we need to make sure that they have their own files in `.husky`.

Reply:
I also think the files need to have headers like this:

```
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"
```

So, the contents of `.husky/pre-commit` would be:

```sh
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

npm run test
```